### DISCUSSION TITLE
Only admin can create teams

### DISCUSSION THREAD

**Discussion by Leon-Luu** [hoppscotch/hoppscotch#3198] [Ideas]
We would like to propose a new feature to restrict new team creations by none-admin person.
--------------------------
In our business we use self-hosted which we would like to restrict users from creating a new team as per default.  
We would like to have a config parameter to disable this.
We would like as Hopscotch admin to be the only person to create a new team.

**Comment by WestFarmer** [comment]
We want this too

**Comment by WestFarmer** [comment]
@liyasthomas  Can you give some comments on this one ?
Do you guys have plan to support or you think it's not a valid feature request.
Thanks!

**Comment by liyasthomas** [comment]
By design, the Hoppscotch app allows every authenticated user to create a team. But here's something that might workout for you;

As far as the Hoppscotch Self-Hosted instances go, we've two editions.

1. Community Edition
2. Enterprise Edition

Both self-hosted editions have an admin panel with features for users and team management. I'd advise you to check out Hoppscotch [Self-Hosted Enterprise Edition](https://hoppscotch.com/pricing) which has the ability to create Private Self Hosted Hoppscotch instances. Only admin-approved users can sign up and log in to Hoppscotch App and thereby create teams.

Imagine the use case for organizations who want to self-host Hoppscotch on their own infrastructure but want to limit their instance to their employees — Hoppscotch Self-Host Enterprise will be a good fit.

Self-hosting documentation: https://docs.hoppscotch.io/documentation/self-host/getting-started
